GNUmakefile

```diff
@@ -49,7 +52,23 @@ pep8: $(PY_FILES)
 		echo "pep8 not installed!  Cannot check PEP8 compliance; aborting." >&2; \
 		exit 1; \
 	fi
-	$(LIST_PY_FILES) | xargs pep8 --ignore=E221,E251,E272,E241,E731
+	$(LIST_PY_FILES) | xargs pep8 --ignore=E221,E241,E251,E272,E731
+
+.PHONY: flake8
+flake8: $(PY_FILES)
+	@if ! which flake8 >/dev/null 2>&1; then \
+		echo "flake8 not installed!  Aborting." >&2; \
+		exit 1; \
+	fi
+	$(LIST_PY_FILES) | xargs flake8 --ignore=E221,E241,E251,E272,E731
+
+.PHONY: pylint
+pylint: $(PY_FILES)
+	@if ! which pylint >/dev/null 2>&1; then \
+		echo "pylint not installed!  Aborting." >&2; \
+		exit 1; \
+	fi
+	$(LIST_PY_FILES) | xargs pylint
```

Comment:
Thanks for your hint but it did not make any difference in my tests on my laptop (8 cores - -j 8). Took always about 5sec.